### DISCUSSION TITLE
LDAP Authentication

### DISCUSSION THREAD

**Discussion by Dozentz** [outline/outline#4218] [Ideas]
Сan be implemented LDAP Authentication ?

**Comment by Limezy** [comment]
You can go through an intermediate tool such as Dex, which will use LDAP as its backend user database and provide you with an OpenID layer, compatible with Outline

**Reply by Limezy** [reply]
https://dexidp.io/docs/connectors/ldap/

**Reply by kranzfr3d** [reply]
Thanks for the input with the OpenID. I am going to give Keycloak a try since its with Admin GUI. Dex does not appear to be suitable for corporate use.
https://hub.docker.com/r/keycloak/keycloak
https://www.keycloak.org/getting-started/getting-started-docker
